pom.xml

```diff
@@ -28,13 +28,13 @@
 
     <!-- Properties -->
     <properties>
-        <version.arquillian>1.1.11.Final</version.arquillian>
-        <version.arquillian.cube>1.0.0.Alpha16</version.arquillian.cube>
+        <version.arquillian>1.1.13.Final</version.arquillian>
+        <version.arquillian.cube>1.6.1-SNAPSHOT</version.arquillian.cube>
```

Comment:
I agree with you, probably we can wait the next release with all changes that we need.
I also (hopefully) found the cause of the hanging tests that has been happening lately:
https://github.com/arquillian/arquillian-cube/issues/752


But, don't you think better first update the arquillian-cube and after all is tests is running ok bump the arquillian version?